From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Shelley Vohr <shelley.vohr@gmail.com>
Date: Fri, 18 Oct 2024 17:01:06 +0200
Subject: fix: remove harmony-import-assertions from node.cc

harmony-import-assertions has been removed from V8 as of
https://chromium-review.googlesource.com/c/v8/v8/+/5507047,
so we should remove it from node.cc as well.

This patch can be removed when we upgrade to a V8 version that
contains the above CL.

diff --git a/src/node.cc b/src/node.cc
index ccc1085a84b214d241687fa9ebd45b55b2cc60d8..1df8e1f00a0e2ffa63bfd4369240b837ab6a9c50 100644
--- a/src/node.cc
+++ b/src/node.cc
@@ -804,7 +804,7 @@ static ExitCode ProcessGlobalArgsInternal(std::vector<std::string>* args,
   }
   // TODO(nicolo-ribaudo): remove this once V8 doesn't enable it by default
   // anymore.
-  v8_args.emplace_back("--no-harmony-import-assertions");
+  // v8_args.emplace_back("--no-harmony-import-assertions");
 
   auto env_opts = per_process::cli_options->per_isolate->per_env;
   if (std::find(v8_args.begin(), v8_args.end(),
